client.go

```diff
@@ -80,8 +80,10 @@ type SnowthClient struct {
 
 // NewSnowthClient - given a variadic addrs parameter, the client will
 // construct all the needed state to communicate with a group of nodes
-// which constitute a cluster.  It will return a pointer to a SnowthClient
-func NewSnowthClient(addrs ...string) (*SnowthClient, error) {
+// which constitute a cluster.  It will return a pointer to a SnowthClient.
+// The discover parameter when true will allow the client to discover new
+// nodes from the topology
+func NewSnowthClient(discover bool, addrs ...string) (*SnowthClient, error) {
```

Comment:
*outofscope* - wouldnt a list of addrs work better ( especially for getting addrs from a config? )

Reply:
I feel like `NewSnowthClient(true, addrs...)` where addrs is a `[]string{}` functionally is the same, plus it makes the case where you enumerate the urls in code look better personally.